invisible_cities/cities/detsim_simulate_electrons_test.py

```diff
@@ -61,7 +60,7 @@ def test_generate_ionization_electrons(MChits_and_detsim_electron_simulation_par
 def test_drift_electrons(MChits_and_detsim_electron_simulation_params):
     _, _, zs, energies,\
     wi, fano_factor, lifetime, drift_velocity,\
-    _, _ = MChits_and_detsim_electron_simulation_params
+    *_ = MChits_and_detsim_electron_simulation_params
```

Comment:
Please indent continued lines. (Not only on line 63, but also on 52 ... and anywhere else they might be.)

Without the indentation, the continuation looks like a line in its own right, so I wasted far too much time staring at 

```python
*_ = just_some_identifier
```
and wondering what the point was. Only when noticing the `\` at the end of the *previous* line did it make any sense. Indentation of the continuation would have made it obvious that it's part of something that precedes it.

Generally speaking, try to avoid continued lines: they are fragile and confusing. Frequently you can replace them with brackets, but not in this case.

In this specific case, you are unpacking something that contains over 8 components and relying on getting the order right. To me this is begging for a `namedtuple` or some other way of naming these components.